### ISSUE TITLE
ImageResize needs fixing in Newpost.vue

### ISSUE THREAD

**Issue by Rajan** [Rajan/lesspod#30]
Uncommenting ImageResize code in main.js (under client/vue/src) and inside Newpost.vue produces errors. There are errors inherent to resize module.

**Comment by anurag-git** [comment]
Rajan,

As per the link kensnyder/quill-image-resize-module#7 fixed this issue. Please cross check.

Anurag

**Comment by Rajan** [comment]
@anurag-git it isn't fixed completely. Kindly uncomment image_resize options in the newpost.vue and see it for yourself that new posts are not being created.

**Comment by anurag-git** [comment]
Yes, I see that, how do see logs to understand what;s wrong. I don't see any logs on my console.

I am working on windows.

**Comment by Rajan** [comment]
@anurag-git did you get a chance to look into this?

**Comment by anurag-git** [comment]
Also it is taking time to understand whole code base. Do you have any kind of documents etc.?

As I can't spend full time on it, after understanding the whole project, I can complete more tasks in less time.

**Comment by sujaykundu777** [comment]
Yeah we need to have a documentation. Any ideas on what blueprints platform we can start on documentation.

**Comment by Rajan** [comment]
@sujaykundu777 Vuejs has good documentation. We should use the same thing. 👍

**Comment by sujaykundu777** [comment]
@Rajan We can use gitbook. Looks Great too.
